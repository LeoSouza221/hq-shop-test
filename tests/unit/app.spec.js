import { mount } from '@vue/test-utils';
import App from '@/App.vue';

test('app match with snapshot', () => {
  const wrapper = mount(App);

  expect(wrapper.element).toMatchSnapshot();
});
