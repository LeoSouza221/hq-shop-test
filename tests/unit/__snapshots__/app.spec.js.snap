// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app match with snapshot 1`] = `
<v-app>
  <v-fade-transition
    mode="out-in"
  >
    <router-view />
  </v-fade-transition>
</v-app>
`;
